src/components/popups/FeaturedPhoto.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import XIcon from '@/components/icons/XIcon.vue';
+
+const prop = defineProps({
+	featuredPhoto: {
+		type: Object,
+		required: true,
+	},
+});
+defineEmits(['close']);
+</script>
+
+<template>
+	<Teleport to="#popup">
+		<div
+			class="popup bg-darkBG dark:bg-gray5 modal-animation fixed top-0 bottom-0 left-0 right-0 z-30 flex h-screen w-full items-center justify-center bg-opacity-50 dark:bg-opacity-50"
+			@click.stop="$emit(`close`)"
+		>
+			<!-- Container -->
+			<div class="w-full lg:w-600 max-h-90 dark:bg-darkBGStop card-animation z-10 overflow-y-auto rounded-lg p-6 pt-4">
```

Comment:
Not sure if we should have the `lg:w-600` on this one as it is supposed to zoom on the featuredPhoto to see it better. Same for the height. can you check on Capsule-vue to see how it is there?